docs/pages/run/validator-management/monitoring-attestation.md

```diff
@@ -0,0 +1,518 @@
+```
+
+### Configure Grafana
+
+Steps:
+
+1. Open at http://localhost:3000
+2. Go to Configuration then Data Sources
+3. Add Prometheus with URL http://localhost:9090
+4. Import or create dashboards
+
+Look for: Correlate events. Peer count drops at 3:17 AM, attestations fail at 3:17:30 - you found it.
+
+## Use beaconcha.in
+
+Your node shows what it thinks. beaconcha.in shows what happened on-chain.
+
+### Enable Monitoring
+
+```bash
+lodestar beacon \
+  --monitoring.endpoint="https://beaconcha.in/api/v1/client/metrics?apikey=YOUR_KEY&machine=my-validator" \
+  --monitoring.interval=60000
+```
+
+Get API key from beaconcha.in settings.
+
+Why do this:
+
+- Mobile alerts for offline validators
+- Historical data
+- Compare with network averages
+- Track earnings
+
+### Check Dashboard
+
+Go to beaconcha.in, search your validator index. You'll see:
+
+- Attestation effectiveness percentage
+- Inclusion distance trends
+- Missed attestations
+- Income
+
+Earning less than network average? Dig deeper. Everyone down? Network-wide issues.
+
+## Effectiveness vs Inclusion Distance
+
+Effectiveness accounts for network conditions, not just speed.
+
+Attestation at slot S, included at slot I.
+
+earliest_possible_inclusion equals E, which is the first slot after S with a block.
+
+Formula: effectiveness = (E - S) / (I - S)
+
+Key point: If next slot had no block because proposer was offline, you're not penalized. Can still hit 100% even with distance greater than 1.
+
+Examples:
+
+- Slot 5, included slot 6: 100% perfect
+- Slot 5, no block slot 6, included slot 7: 100% not your fault
+- Slot 5, block at slot 6, included slot 7: 50% you were slow
+
+Check beaconcha.in for intervening slots.
+
+## Common Issues
+
+### Low Peer Count
+
+Seeing: Under 20 peers
+
+Fix:
+
+- Check firewall, open LibP2P ports
+- Port forwarding if behind NAT
+- Add bootstrap nodes with --network.bootMultiaddrs
+- Increase max peers with --network.maxPeers=100
+
+### High Resource Usage
+
+Seeing: CPU above 80%, memory climbing, high iowait
+
+Quick fix:
+
+- Restart Lodestar
+- Stop other services
+- Check for known bugs
+
+Long-term:
+
+- Upgrade hardware, more RAM and SSD
+- Use NVMe instead of SATA
+- Give more CPU cores
+
+### Clock Issues
+
+Seeing: is_syncing true won't clear, slot number off
+
+Fix:
+
+- Enable NTP with sudo timedatectl set-ntp true
+- Check with timedatectl status
+- Restart Lodestar
+
+### Behind on Sync
+
+Seeing: beaconchain_current_slot way behind network
+
+Fix:
+
+- Wait for sync
+- Check execution layer synced
+- Verify bandwidth for blocks
+- Check disk I/O
+
+## Debug Logging
+
+For deep investigation, run with debug logs:
+
+```bash
+lodestar beacon --logLevel=debug 2>&1 | tee /var/log/lodestar/debug.log
+```
+
+Or with systemd:
+
+```bash
+sudo journalctl -u lodestar.service -f
+```
+
+Note: Debug logs are verbose. Only use when troubleshooting. Set up log rotation so disk doesn't fill.
+
+## Regular Maintenance
```

Comment:
What's the rationale for telling people. how to manage their own regular maintenance?